lib/hon_graffiti_phoenix/parsers/decorated_string.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HonGraffitiPhoenix.Parsers.DecoratedString do
+  @moduledoc """
+  Provides a struct for strings with hon style markdown
```

Comment:
Gonna make literally the most annoying request: can you change this to `markup`?